workers/vercel-edge-functions/prepare.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export PRISMA_TELEMETRY_INFORMATION='e2e-tests data-proxy vercel-edge-functions build'
+
+source ../../utils/crypto/envVars.sh VERCEL_DATA_PROXY_URL VERCEL_EDGE_FUNCTIONS_PROJECT_ID VERCEL_EDGE_FUNCTIONS_ORG_ID
+
+# we set the vercel project & organization environment variables
+export VERCEL_PROJECT_ID=$VERCEL_EDGE_FUNCTIONS_PROJECT_ID
+export VERCEL_ORG_ID=$VERCEL_EDGE_FUNCTIONS_ORG_ID
```

Comment:
Is this a different org than our other proejects?

Reply:
Nope, should I use the env var instead?